examples/credit-risk-end-to-end/01_Credit_Risk_Data_Prep.ipynb

```diff
@@ -0,0 +1,1103 @@
+       "age                                                 27                  24   \n",
+       "other_payment_plans                               none                none   \n",
+       "housing                                            own                rent   \n",
+       "existing_credits                                     2                   1   \n",
+       "job                                            skilled             skilled   \n",
+       "num_dependents                                       1                   1   \n",
+       "own_telephone                                      yes                none   \n",
+       "foreign_worker                                     yes                 yes   \n",
+       "class                                             good                 bad   \n",
+       "\n",
+       "                                   454                        453  \n",
+       "checking_status                     <0                no checking  \n",
+       "duration                            24                         24  \n",
+       "credit_history           existing paid              existing paid  \n",
+       "purpose                        new car                   used car  \n",
+       "credit_amount                     4817                       2670  \n",
+       "savings_status                    <100                       <100  \n",
+       "employment                      4<=X<7                        >=7  \n",
+       "installment_commitment               2                          4  \n",
+       "personal_status            male single                male single  \n",
+       "other_parties             co applicant                       none  \n",
+       "residence_since                      3                          4  \n",
+       "property_magnitude      life insurance                        car  \n",
+       "age                                 31                         35  \n",
+       "other_payment_plans               none                       none  \n",
+       "housing                            own                        own  \n",
+       "existing_credits                     1                          1  \n",
+       "job                            skilled  high qualif/self emp/mgmt  \n",
+       "num_dependents                       1                          1  \n",
+       "own_telephone                      yes                        yes  \n",
+       "foreign_worker                     yes                        yes  \n",
+       "class                              bad                       good  "
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# View first records in training data (transposed)\n",
+    "train.head().T"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6825a3ed-2495-4ded-8f5f-1c64041ba5ac",
+   "metadata": {},
+   "source": [
+    "### Feature Engineering"
```

Comment:
I don't see any actual feature engineering here, right? Or did I miss something?

Reply:
Pretty much right. In the example I explain this as we are training a baseline model, and there is opportunity to improve by doing more feature engineering. I throw away the categorical features and keep the numerical; then change the outcome from text to an indicator variable. There's certainly an opportunity to do more; I would just want to make sure we keep the simple feel.

Reply:
I would recommend adding even a trivial feature calculation to highlight it.

Reply:
I added a couple of ordinal features (converted from categorical columns)

Reply:
As @tokoko mentioned I think we should use a different featurization example as one hot encoding or ordinal encoding often happens outside of Feast during an inference pipeline (though we should build for this in the future to handle the important implementation detail of new categories for online serving). 

I think a ratio or product of two other features is sufficient to highlight the goal.